README.rst

```diff
@@ -39,24 +50,25 @@ Client Features
 
   * Full read/write protocol on discrete and register
   * Most of the extended protocol (diagnostic/file/pipe/setting/information)
-  * TCP, UDP, Serial ASCII, Serial RTU, and Serial Binary
-  * asynchronous(powered by twisted/tornado/asyncio) and synchronous versions
+  * TCP, RTUOVERTCP, UDP, Serial ASCII, Serial RTU, and Serial Binary
+  * asynchronous(powered by asyncio/twisted/tornado) and synchronous versions
   * Payload builder/decoder utilities
   * Pymodbus REPL for quick tests
+  * Customable framer to allow for custom implementations
 
 ~~~~~~~~~~~~~~~~~~~~
 Server Features
 ~~~~~~~~~~~~~~~~~~~~
 
   * Can function as a fully implemented modbus server
-  * TCP, UDP, Serial ASCII, Serial RTU, and Serial Binary
+  * TCP, RTUOVERTCP, UDP, Serial ASCII, Serial RTU, and Serial Binary
```

Comment:
Can we change `RTUOVERTCP` to `RTU-OVER-TCP` for the sake of readability

Reply:
done in all places.